dll/win32/fmifs/format.c

```diff
@@ -65,20 +64,16 @@ FormatEx(
         return;
     }
 
-#if 1
-    DPRINT1("Warning: use GetVolumeNameForVolumeMountPointW() instead!\n");
-    swprintf(VolumeName, L"\\??\\%c:", towupper(DriveRoot[0]));
-    RtlCreateUnicodeString(&usDriveRoot, VolumeName);
-    /* Code disabled as long as our storage stack doesn't understand IOCTL_MOUNTDEV_QUERY_DEVICE_NAME */
-#else
     if (!GetVolumeNameForVolumeMountPointW(DriveRoot, VolumeName, RTL_NUMBER_OF(VolumeName)) ||
-        !RtlDosPathNameToNtPathName_U(VolumeName, &usDriveRoot, NULL, &CurDir))
+        !RtlDosPathNameToNtPathName_U(VolumeName, &usDriveRoot, NULL, NULL))
```

Comment:
One potential suggestion I was thinking about while also looking at this piece of code (due to my recent investigations on the mountmgr):
Wouldn't it be of interest, to fall back to using the drive letter, in case the `GetVolumeNameForVolumeMountPointW` function fails? But maybe keep the letter in the Win32 format, so that, by passing it to `RtlDosPathNameToNtPathName_U`, it gets converted to the expected `\??\x:` format.

Reply:
While testing this I also noticed that this code won't handle the case when `DriveRoot` is already a Volume GUID path, which is probably something I want to fix within this PR too. I can also implement your suggestion while at it, although I am not sure if it makes sense to attempt formatting something that volume GUID path cannot be even obtained for?